src/routes/(app)/+layout.svelte

```diff
@@ -476,7 +476,7 @@
 
 		<footer class="footer h-auto items-center flex print:hidden">
 			<div
-				class="mx-auto max-w-7xl px-6 py-6 flex items-start justify-between gap-2 gap-y-8 w-full flex-wrap"
+				class="mx-auto max-w-7xl px-6 py-6 grid items-start justify-between gap-y-8 w-full grid-cols-1 lg:grid-cols-4 gap-4"
```

Comment:
`lg` is `1024px`

Anyway let's keep the behavior the same for large screens? You can do things like `grid grid-cols-1 lg:flex lg:flex-wrap`